gym/spaces/multi_binary.py

```diff
@@ -3,6 +3,17 @@
 
 
 class MultiBinary(Space):
+    '''
+    An n-dimensional binary space. 
+
+    The argument to MultiBinary defines n.
+    
+    Example Usage:
+    
+    self.observation_space = spaces.MultiBinary(5)
+    
```

Comment:
looks good, but could you also add an example of what would self.observation_space.sample() return?

Reply:
@pzhokhov made the changes.